tests/src/whisk/core/controller/test/MetaApiTests.scala

```diff
@@ -853,6 +853,33 @@ class MetaApiTests extends ControllerTestCommon with WhiskMetaApi with BeforeAnd
                     }
                 }
         }
+
+        it should s"invoke action with options verb (auth? ${creds.isDefined})" in {
+            implicit val tid = transid()
+
+            Seq(s"$systemId/default/export_c.json").
+              foreach { path =>
+                  Options(s"$testRoutePath/$path") ~> sealRoute(routes(creds)) ~> check {
```

Comment:
This test should mock a response from an options action and check that the headers are set accordingly - there is on other test in this suite that checks headers as an example to follow.

We might also add a test in `WskWebActionsTests` to confirm the options response is from the action, and NOT from the top level option in the controller.